Source/GMCAbilitySystem/Private/Effects/GMCAbilityEffect.cpp

```diff
@@ -65,12 +65,23 @@ void UGMCAbilityEffect::StartEffect()
 	AddTagsToOwner();
 	AddAbilitiesToOwner();
 
-	// Duration and Instant applies immediately.
+	// Instant effects modify base value and end instantly
+	if (EffectData.bIsInstant)
```

Comment:
now that we have this we should prob delete the repeated if (EffectData.bIsInstant) check below. also return in this if block since we are ending an effect and don't want to move through the function further